DAQ_System/Source/main.cpp

```diff
@@ -1,23 +1,140 @@
 #include "mbed.h"
-#include <iostream>
+#include <cstdio>
+#include <cstdlib>
+
 
 #include "L3GD20H.h"
 // Found in: https://os.mbed.com/users/julioefajardo/code/L3GD20H/docs/tip/classL3GD20H.html
 
 
+#define sensitivity (.00875 )// this is using a sensitivity of 250 dps which is equivalent to .00875
+#define DPS_TO_RadS (3.14159/180) // this is the conversion factor for degrees per second to radians per second all you have to do is multiply the callibrated data or the raw data by the conversion factor
+L3GD20H gyro(I2C_SDA, I2C_SCL);
+
+void averageOffsetCalc(short averageOffset[], short sum[]);
+void degreesPerSecondConversion(short degreesPerSecond[],short offset[], short gyroData[]);
+void radiansPerSecondConversion(float radiansPerSecond[],short degreesPerSecond[]);
```

Comment:
Tiny: Let's do Pascal case for functions (just make the first letter capitalized)

Nice attention to detail using floats and shorts!